lxd/device/nic_bridged.go

```diff
@@ -113,6 +113,16 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 			if dhcpv4Subnet != nil && !dhcpalloc.DHCPValidIP(dhcpv4Subnet, nil, net.ParseIP(d.config["ipv4.address"])) {
 				return fmt.Errorf("Device IP address %q not within network %q subnet", d.config["ipv4.address"], n.Name())
 			}
+
+			ip, _, err := net.ParseCIDR(netConfig["ipv4.address"])
```

Comment:
What happens when `ipv4.address` isn't set?

Reply:
It can be empty or set to `none` on a network.

Reply:
Ah yeah, there's a case where that's allowed with `security.ipv4_filtering` set to true, which I missed.